root/static/scripts/common/components/WikipediaExtract.js

```diff
@@ -38,7 +38,7 @@ class WikipediaExtract extends React.Component<Props, State> {
     }
   }
 
-  render() {
+  render(): React.MixedElement | null {
```

Comment:
Why not `React.Element<typeof React.Fragment> | null` instead?

Reply:
Same as above, that type information isn't used by anything.